src/sys_status.h

```diff
@@ -1,15 +1,15 @@
 #ifndef SYS_STATUS_H
 #define SYS_STATUS_H
 
-struct systemStatus_structure {                     // currently 14 bytes long
+struct systemStatus_structure {  
   uint8_t structuresVersion;                        // Version of the data structures (system and data)
   uint8_t placeholder;                              // available for future use
   bool clockSet;                                    // Tells us if we need to connect and set the RTC
-  uint8_t connectedStatus;
-  uint8_t verboseMode;
-  uint8_t solarPowerMode;
-  uint8_t lowPowerMode; // bool?
-  uint8_t lowBatteryMode;
+  bool connectedStatus;                             // Are we supposed to be connected?
+  bool verboseMode;                                 // Turns on extra messaging
+  bool solarPowerMode;                              // Powered by a solar panel or utility power
+  bool lowPowerMode;                                // Does the device need to run disconnected to save battery
```

Comment:
This more strict typing is great! And it paid off instantly by allowing you to eliminate a lot of the system value checks!